app/assets/stylesheets/static.scss

```diff
@@ -0,0 +1,20 @@
+.site {
+  background: $orange;
```

Comment:
the proper way of doing the orange top border is to use the `border-top` property, also 20px is way too big.